pupepat/ellipse.py

```diff
@@ -67,3 +67,24 @@ def generate_ellipse(x0, y0, a, b, theta):
     np.append(x, x[0])
     np.append(y, y[0])
     return x, y
+
+
+def calculate_donut_offset(inner_coordinate, outer_coordinate, sbig=False):
```

Comment:
I know this function is overkill, but it gives a good location for documentation about the M1 offset conversion. and could potentially be useful if this becomes a thing for other instruments on other telescopes?